extensions/guacamole-auth-sso/modules/guacamole-auth-sso-base/src/main/resources/translations/it.json

```diff
@@ -0,0 +1,23 @@
+{
+
+    "DATA_SOURCE_CAS" : {
+        "NAME" : "CAS SSO Backend"
```

Comment:
You can leave out any strings that you're not specifically translating, unless this is the Italian translation?

Reply:
Done, hope it's ok now